Makefile

```diff
@@ -132,28 +133,19 @@ test: test-unit test-e2e
 test-unit:
 	go test -race -short $(PKGS) -count=1
 
+.PHONY: test-rules
+test-rules: $(PROMTOOL_BIN) $(GOJSONTOYAML_BIN) $(ASSETS)
+	@promtool check rules <(cat assets/prometheus-k8s/rules.yaml | $(GOJSONTOYAML_BIN) -yamltojson | jq .spec) | tee tmp/rules.out
+	@grep duplicate tmp/rules.out && exit 1
```

Comment:
Do we really want to error when we find duplicated rules? AFAIK it is only an indication from promtool and in some cases, it is valid to have duplicates. Like with `code_verb:apiserver_request_total:increase30d`

Reply:
I think we do, especially for rules we are shipping.